src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -16,33 +17,61 @@ export interface AdminVenuesProps {
 }
 
 export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
+  const [
+    currentSortingOption,
+    setCurrentSortingOption,
+  ] = useState<VenueSortingOptions>(VenueSortingOptions.az);
+
+  const sortedVenues = useMemo(() => {
+    return sortVenues(venues, currentSortingOption) ?? [];
+  }, [currentSortingOption, venues]);
+
   const renderedPartyVenues = useMemo(
     () =>
-      venues
+      sortedVenues
         ?.filter(isPartyMapVenue)
         .map((venue) => <AdminVenueCard key={venue.id} venue={venue} />),
-    [venues]
+    [sortedVenues]
+  );
+
+  const sortingOptions = useMemo(
+    () => (
+      <DropdownButton variant="secondary" title="Sort venues">
+        {Object.values(VenueSortingOptions).map((sortingOption) => (
+          <Dropdown.Item
+            key={sortingOption}
+            onClick={() => setCurrentSortingOption(sortingOption)}
+          >
+            {sortingOption}
+          </Dropdown.Item>
+        ))}
+      </DropdownButton>
+    ),
+    []
   );
 
   const hasVenues = renderedPartyVenues.length > 0;
 
   return (
-    <div className="admin-venue">
-      <div className="admin-venue__header">
-        <div className="admin-venue__title">Admin Dashboard</div>
+    <div className="AdminVenues">
+      <div className="AdminVenues__header">
+        <div className="AdminVenues__header--left">
+          <div className="AdminVenues__header-title">Admin Dashboard</div>
+          {sortingOptions}
+        </div>
         <Button as={Link} to="/admin-ng/create/venue">
           Create a new space
         </Button>
       </div>
       <div
-        className={classNames("admin-venue__cards", {
-          "admin-venue__cards--empty": !hasVenues,
+        className={classNames("AdminVenues__cards", {
+          "AdminVenues__cards--empty": !hasVenues,
         })}
       >
         {!hasVenues && (
           <>
-            <div className="admin-venue__title">Welcome!</div>
-            <div className="admin-venue__title">
+            <div className="AdminVenues__title">Welcome!</div>
+            <div className="AdminVenues__title">
```

Comment:
Two titles? 

Does `< br/ >` inside a single title `<div >` work the same?

Reply:
Nope, flex won't center bot text if they are using <br/> because it's a single element. I suppose the name the style can be improve and changed to `header` or `welcome-message`